### PR TITLE
Update docs example for Subscription

### PR DESCRIPTION
**Pull request by Fedos**

### CODE REVIEW

File: Sources/SwiftGraphQLCodegen/Generator.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ public struct GraphQLCodegen {
     func generate(schema: Schema) throws -> String {
         let code = """
         // This file was auto-generated using maticzav/swift-graphql. DO NOT EDIT MANUALLY!
+        import Foundation
```
Comment (Fedos) [inline Sources/SwiftGraphQLCodegen/Generator.swift:36]: BTW, this one was needed for custom scalars

File: website/docs/querying.mdx [hunk 2]
```diff
@@ -33,15 +33,23 @@ let subscription = Selection.Subscription {
...
+cancellable = socket.subscribe(to: subscription) { result in
     if let data = try? result.get() {
         print(data)
     }
 }
 
-// To close the socket.
-task.close()
+// Close the socket.
+cancellable = nil
```
Comment (Amzd) [inline website/docs/querying.mdx:52]: or cancellable?.cancel() which is more readable imo

**Review by Amzd — APPROVED** [review]

File: website/docs/querying.mdx (hunk 2 above)
Comment (Fedos) [reply website/docs/querying.mdx:52]: AFAIK it cancels it's publisher before deinitialization, the point is to close the socket as well

Comment (Fedos) [reply website/docs/querying.mdx:52]: I doubled it from the StarWars example, let me dive deeper into how it works...

Comment (Fedos) [reply website/docs/querying.mdx:52]: My guess that there is also missing explicit `socket?.stop()` after that

**Comment by maticzav** [comment]
Thank you for opening this PR! The new version introduces different handling of WebSockets, and this example isn't longer up to date. I'll close the PR, but I am thankful you took the time to fix it.

**Pull request closed without merging**